### PR TITLE
Filip/update readme 2

### PR DESCRIPTION
**Pull request by filiplazovic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,153 +1,141 @@
...
 
 ## Anchor
 
-Anchor protocol is very similar to mixer. Instead of proving that the membership inside one merkle tree, we are proving the membership in many merkle trees. These trees can live on many different blockchains, and if the merkle tree states are synced across chains, this will allow us to make cross-chain anonymous transactions. Higher level overview of how Anchor works:
+Anchor protocol is very similar to the mixer. Instead of proving the membership inside one Merkle tree, we are proving the membership in many Merkle trees. These trees can live on many different blockchains, and if the Merkle tree states are synced across chains, this will allow us to make cross-chain anonymous transactions. Higher-level overview of how Anchor works:
```
Comment (drewstone) [inline README.md:100]: Proving inclusion in one of many

File: README.md [hunk 2]
```diff
@@ -1,153 +1,141 @@
...
 
-1. We are computing the leaf commitment using the Poseidon hash function by passing: `secret` (private input), `nullifier` (private input).
-2. We are computing the nullifier hash using the Poseidon hash function, by passing: `nullifier` (private input)
+1. We are computing the leaf commitment using the Poseidon hash function passing: `secret` (private input), `nullifier` (private input).
```
Comment (drewstone) [inline README.md:102]: Shouldn't this include the chain Id for the anchor leaf hash computation too?

**Review by drewstone — CHANGES_REQUESTED** [review]
Looks good. Left some comments. Can we include links to the relayer repo as well? Are there scripts we can also link for generating trusted setup parameters for plonk and R1CS for these circuits? I think that would be helpful to have here.

**Comment by filiplazovic** [comment]
Fixed. There is also a task for trusted setups, which has much more context: https://github.com/webb-tools/arkworks-gadgets/issues/78

**Pull request merged**